cds/modules/previewer/static/css/cds_previewer/video.css

```diff
@@ -7,6 +7,7 @@ html, body {
   font-family: 'PT Sans', sans-serif;
 }
 
+.video-js,
```

Comment:
just to be sure 100%, is the forcing of w/h working in both case when fluid: true and false? Not breaking anything?

Reply:
This is just the actual player to fill the 100% of the previewer page and not the embed iframe :)